recipes/mplfinance/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mplfinance" %}
+{% set version = "0.12.7a0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3e7a6f80a0386419b184434dfc74a943f55f86baa906c25fe2a8d338d5edce62
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3.6
+  host:
+    - matplotlib
+    - pandas
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib
+    - pandas
+    - python >=3.6
+
+test:
+  imports:
+    - mplfinance
+    - mplfinance._styledata
+
+about:
+  home: "http://github.com/matplotlib/mplfinance"
+  license: BSD
+  license_family: BSD
```

Comment:
Is this a BSD license or a PSF?

https://spdx.org/licenses/PSF-2.0.html

It looks similar to the one included with matplotlib and this is how it is listed in that recipe:

https://github.com/conda-forge/matplotlib-feedstock/blob/551e6c0dab27b4c09f08f9ff43647945c3d85ef3/recipe/meta.yaml#L106-L109

Reply:
The license was copied from the matplotlib license file (which as I understand it is "BSD-like") ... with a couple tweaks (changing matplotlib to mplfinance).
I'd be happy to make the license a more standard license, however I don't know if there are any legalities associated with changing the license.  The mplfinance packaged already existed (but was unmaintained for 3 years) when I became the maintainer about a year ago.

Reply:
I've updated the license info to more closely resemble matplotlib's meta.yaml license info.